ivy/data_classes/array/experimental/manipulation.py

```diff
@@ -231,6 +229,12 @@ def rot90(
 
         Parameters
         ----------
+        copy
+            boolean indicating whether or not to copy the input array.
+            If True, the function must always copy.
+            If False, the function must never copy.
+            In case copy is False we avoid copying by returning
+             a view of the input array.
```

Comment:
Could you move this to under `self`? We want the docstrings to be in the same order as the arguments.